src/hotspot/share/gc/shenandoah/shenandoahVerifier.cpp

```diff
@@ -68,6 +61,7 @@ class ShenandoahVerifyOopClosure : public BasicOopIterateClosure {
   ShenandoahLivenessData* _ld;
   void* _interior_loc;
   oop _loc;
+  ReferenceIterationMode _ref_mode;
```

Comment:
I don't see where this new field is read.

Reply:
Okay, I see now that `reference_iteration_mode` overrides a virtual method defined in `OopIterateClosure`. Perhaps mark it with `override` for readability?

Reply:
Right. Since `override` is all-or-nothing, I had to add it to other methods as well.